Codecademy/client-modules
# feat: add a illustration for business team

### Overview


This PR aims to add a new illustration for the business team. 


### PR Checklist

- [x] Related to designs: https://www.figma.com/file/Duc8xMrpR50jagRMGKDLWy/Welcome?node-id=266%3A6576
- [x] Related to JIRA ticket: EN-498



1. Visit https://6030466796063b4fc1b7e3f3--gamut-preview.netlify.app/storybook/?path=/docs/atoms-illustrations--editable-illustration
2. Change `illustration` `Control` to `Onboarding` and see that the illustration matches the designs.
